frontend/src/components/dashboard/Dashboard.jsx

```diff
@@ -39,8 +39,12 @@ const charts2 = [
 ];
 
 export default function Dashboard() {
+  // const isSelectedUI = JobResultSections.VISUALIZER;
   const { guideState, setGuideState } = useGuideContext();
 
+  const [orgstate, setorgState] = useState(() => false);
```

Comment:
It's a very silly suggestion but I'd to have orgState, it has the same style of guideState (it's strange the linter doesn't mark this)